bridges/CssSelector2Bridge.php

```diff
@@ -0,0 +1,421 @@
+<?php
+
+class CssSelector2Bridge extends BridgeAbstract
+{
+    const MAINTAINER = 'Lars Stegman';
+    const NAME = 'CSS Selector 2 Bridge';
+    const URI = 'https://github.com/RSS-Bridge/rss-bridge/';
+    const DESCRIPTION = <<<EOT
+        Convert any site to RSS feed using CSS selectors (Advanced Users). The bridge first selects 
+        the element describing the article entries. It then extracts the links to the articles from 
+        these elements. If then, depending on the setting "Load article from page", either parses 
+        the selected elements, or downloads the page for each article and parses those. Parsing the 
+        elements or page is done using the provided selectors.
+        EOT;
+    const PARAMETERS = [
+        [
+            'home_page' => [
+                'name' => 'Site URL: Page with latest articles',
+                'exampleValue' => 'https://example.com/blog/',
+                'required' => true
+            ],
+            'cookie' => [
+                'name' => '[Optional] Cookie',
+                'title' => <<<EOT
+                Use when the website does not send the page contents, unless a cookie is included.
+                EOT,
+                'exampleValue' => 'sessionId=deadb33f'
+            ],
+            'entry_element_selector' => [
+                'name' => 'Selector for article entry elements',
+                'title' => <<<EOT
+                This bridge works using CSS selectors, e.g. "div.article" will match all 
+                <div class="article">...</div> on home page, each one being treated as a feed item.
+
+                Use the URL selector option to select the `a` element with the
+                `href` to the article link. If this option is not configured, the first encountered 
+                `a` element is used.
+                EOT,
+                'exampleValue' => 'div.article',
+                'required' => true
+            ],
+            'url_selector' => [
+                'name' => 'Selector for link elements',
```

Comment:
```suggestion
                'name' => '[Optional] Selector for link elements',
```
(Looks like that setting is optional)